src/workingtitle-vcockpits-instruments-navsystems/html_ui/Pages/VCockpit/Instruments/NavSystems/AS1000/MFD/AS1000_MFD_Custom.css

```diff
@@ -0,0 +1,145 @@
+@font-face {
```

Comment:
What's your thinking behind having these as separate files from the main ones?  Is it just to avoid messing with the stock elements as much as possible?   I  haven't been too worried about that, since like I said, we're fixing stock anyway...   but this could make our lives easier especially when it comes to them updating things.   And if by doing this you don't have to commit an entire CSS file in one go that's also a win.

I *think* I'm OK with this, but I reserve the right to change my mind on that.  :D   Confirm your logic here and I think we can talk about it with the group on Slack.  We're trying to maintain a somewhat consistent paradigm in how we work across projects.